cmd/emailreport/README.md

```diff
@@ -0,0 +1,15 @@
+# PnP failure Report tool
+
+This tool reports by email the obtained PnP results from `checkmetrics`
+execution, next it will send the results by email.
+The configuration of SMTP server and checktool arguments are given
+by a configuration file in TOML format, this section includes an
+example of that.
```

Comment:
the intention is to indicate that there is an example of configuration file for email tool in this directory however I am not sure it it worth to mentioned or it could be implicit?